.ci/.e2e-tests-daily.yaml

```diff
@@ -1,14 +1,26 @@
 ---
 SUITES:
+  - suite: "helm"
+    provider: "docker"
+    scenarios:
+      - name: "APM Server"
+        tags: "apm-server"
+        platforms: ["debian_10_amd64"]
+      - name: "Filebeat"
+        tags: "filebeat"
+        platforms: ["debian_10_amd64"]
+      - name: "Metricbeat"
+        tags: "metricbeat"
+        platforms: ["debian_10_amd64"]
   - suite: "fleet"
     scenarios:
-      - name: "Fleet"
-        tags: "fleet_mode"
-        platforms: ["ubuntu_22_04_amd64"]
+#      - name: "Fleet"
+#       tags: "fleet_mode"
+#       platforms: ["ubuntu_22_04_amd64"]
 #         platforms: ["centos8_arm64", "centos8_amd64", "debian_10_arm64", "debian_10_amd64", "sles15", "debian_11_amd64", "ubuntu_22_04_amd64", "windows2019"]
-#       - name: "Integrations"
-#         tags: "integrations"
-#         platforms: ["debian_11_amd64"]
+       - name: "Integrations"
+         tags: "integrations"
+         platforms: ["debian_10_amd64"]
```

Comment:
The change in daily scenarios does not seem related to the changes. Is it correct?

Reply:
Hi Ivan,

No its not related to the current changes, Just to run the tests at my end and validate the changes as working fine.